release-notes/docker-image-reference.md

```diff
@@ -2,23 +2,19 @@
 
 In order for Docker containers to run dotnet cli commands, the sdk that you use to build your project with locally must match the dotnet sdk version inside your container. If you want a lighter-weight container and don’t need to use the cli tools inside the container, then you can pull the runtime images.
 
-|    Docker Images   2/07/17             |    .NET Core   Runtime version in image   |    .NET Core SDK   version in image    (NOTE: All SDK versions can target all   runtimes)    |    Full SDK   version number    |
+|    Docker Images   03/08/17            |    .NET Core   Runtime version in image   |    .NET Core SDK version in image    (NOTE: All SDK versions can target all   runtimes)      |    Full SDK   version number    |
```

Comment:
Sure, but isn't that the 4th column. The 3rd column doesn't seem useful to me (in fact, I think it is confusing).